File: server/scratch/count_docs.py
import os
from pymongo import MongoClient
from dotenv import load_dotenv

# Load environment variables
load_dotenv(dotenv_path=os.path.join(os.path.dirname(__file__), '../server/.env'))

MONGO_URI = os.getenv("MONGO_URI")
if not MONGO_URI:
    print("MONGO_URI not found.")
    exit(1)

client = MongoClient(MONGO_URI)
db = client.get_database()

collections = ['blogs', 'products', 'categories', 'pagecontents']
for coll_name in collections:
    count = db[coll_name].count_documents({})
    print(f"{coll_name}: {count}")

client.close()

File: server/scratch/count_docs.js
const mongoose = require('mongoose');
const dotenv = require('dotenv');
const path = require('path');

dotenv.config({ path: path.join(__dirname, '../.env') });

const MONGO_URI = process.env.MONGO_URI;

async function count() {
  try {
    await mongoose.connect(MONGO_URI);
    const collections = ['blogs', 'products', 'categories', 'pagecontents'];
    for (const coll of collections) {
      const count = await mongoose.connection.db.collection(coll).countDocuments({});
      console.log(`${coll}: ${count}`);
    }
  } catch (err) {
    console.error(err);
  } finally {
    await mongoose.connection.close();
  }
}

count();
